src/Microsoft.DotNet.Build.Tasks/PackageFiles/IL.targets

```diff
@@ -43,6 +43,7 @@
       <_IlasmSwitches Condition="'$(DebugType)' == 'Impl'">$(_IlasmSwitches) -DEBUG=IMPL</_IlasmSwitches>
       <_IlasmSwitches Condition="'$(DebugType)' == 'PdbOnly'">$(_IlasmSwitches) -DEBUG=OPT</_IlasmSwitches>
       <_IlasmSwitches Condition="'$(Optimize)' == 'True'">$(_IlasmSwitches) -OPTIMIZE</_IlasmSwitches>
+      <_IlasmSwitches Condition="'$(IlasmResourceFile)' != ''">$(_IlasmSwitches) -RESOURCES=$(IlasmResourceFile)</_IlasmSwitches>
```

Comment:
I would add the same switch to https://github.com/dotnet/coreclr/blob/master/src/.nuget/Microsoft.NET.Sdk.IL/targets/Microsoft.NET.Sdk.IL.targets#L79

Reply:
Sure we can do that. Out of curiosity, which managed assembly on coreclr gets built using ilasm? Just asking since I presume they would have the same issue where version headers are not built in.

Reply:
CoreCLR has some 4k test projects, which haven't been migrated to use the `ilproj msbuild-sdk` yet.

CoreFX has `S.R.CS.Unsafe`, which is being ported to use the linked targets in https://github.com/dotnet/corefx/pull/31512

Reply:
I see, that makes sense. I'll add this change there as well then.